src/BloomBrowserUI/bookEdit/js/editViewFrame.js

```diff
@@ -137,10 +110,154 @@ function initializeReaderSetupDialog() {
+    parentElement.localizationManager.loadStrings(getAddPageDialogLocalizedStrings(), null, function() {
+
+        var title = parentElement.localizationManager.getText('AddPageDialog.Title', 'Add Page...');
+        var addButtonText = parentElement.localizationManager.getText('AddPageDialog.AddPageButton', 'Add This Page');
+        var descriptionLabel = parentElement.localizationManager.getText('AddPageDialog.DescriptionLabel', 'Description');
+        var blankPreviewMsg = parentElement.localizationManager.getText('AddPageDialog.PreviewMessage',
+            'This will contain a preview of a template page when one is selected.');
+        var dialogContents = FindOrCreateAddPageDiv(templatesJSON, descriptionLabel, blankPreviewMsg);
+
+        // Set dialog size (borrowed from reader tools setup dialog; how much of this is useful?!)
+        var h = 750;
+        var w = 940;
+
+        // This height and width will fit inside the "800 x 600" settings
+        var sw = document.body.scrollWidth;
+        if (sw < 583) {
+            h = 460;
+            w = 390;
+        }
+
+        // This height and width will fit inside the "1024 x 586 Low-end netbook with windows Task bar" settings
+        else if ((sw < 723) || (window.innerHeight < 583)) {
+            h = 460;
+            w = 580;
+        }
+
+        theDialog = $(dialogContents).dialog({
+            autoOpen: false,
+            modal: true,
+            width: w,
+            height: h,
+            position: {
+                my: "left top", at: "left top", of: window
+            },
+            title: title,
+            buttons: {
+                OK: {
+                    text: addButtonText,
+                    icons: {
+                        primary: "ui-icon-plusthick"
+                    },
+                    click: function () {
+                        document.getElementById('addPage_frame').contentWindow.postMessage('AddSelectedPage', '*');
+                    }
+                },
+                Cancel: {
+                    text: parentElement.localizationManager.getText('Common.Cancel', 'Cancel'),
+                    click: function () {
+                        $(this).dialog("close");
+                    }
+                }
+            },
+            close: function() {
+                $(this).remove();
+                fireCSharpEvent('setModalStateEvent', 'false');
+            },
+            open: function () {
+                adjustAddPageButton(addButtonText);
+                setTimeout(function() {
+                    setDialogInnerIframeWidth();
+                }, 200);
+            }
+        });
+        fireCSharpEvent('setModalStateEvent', 'true');
+        theDialog.dialog('open');
+    });
+}
+
+function adjustAddPageButton(addButtonText) {
+    var button = $('.ui-dialog-buttonpane').find('button:contains('+addButtonText+')');
+    button.width(button.width() + 15);
+}
```

Comment:
15 deserves comment or (probably better) a name...is it related to any other width(s)?